test/image/test_vector_image.py

```diff
@@ -9,21 +9,22 @@
 
 
 HERE = Path(__file__).resolve().parent
+SVG_SRCDIR = HERE.parent / "svg/svg_sources"
```

Comment:
```suggestion
SVG_SRC_DIR = HERE.parent / "svg/svg_sources"
```

Reply:
Either version looks plenty clear enough to me.